guide/src/SUMMARY.md

```diff
@@ -0,0 +1,25 @@
+# Summary
+
+[Introduction](README.md)
+
+- [Installation and Usage](installation_and_usage.md)
+- [Configuration](configuration.md)
+- [Clippy's Lints](lints/README.md)
+    - [Cargo]()
+    - [Complexity]()
+    - [Correctness]()
+    - [Pedantic]()
+    - [Perf]()
+    - [Restriction]()
+    - [Style]()
+    - [Deprecated]()
+    - [Nursery]()
+- [Development](development/README.md)
+    - [Basics](development/basics.md)
+    - [Adding Lints](development/adding_lints.md)
+    - [Common Tools](development/common_tools_writing_lints.md)
+    - [Backporting Changes](development/backport.md)
+    - [Updating the Changelog](development/changelog_update.md)
+    - [Release a New Version](development/release.md)
+
+[2021 Roadmap](roadmap-2021.md)
```

Comment:
I hope we'll do more roadmaps in the future. So I would put this in it's own category:

```suggestion
- [Roadmap](...)
    - [2021](roadmap-2021.md)
```

Reply:
I added a roadmap section for these.